commands/build/README.md

```diff
@@ -23,4 +23,5 @@ Options:
   --sourcemap, -m  Generate source map                 [boolean] [default: true]
   --mode           Explicitly set mode
                                  [string] [choices: "production", "development"]
+  --core           Set a core build target            [string] [default: "core"]
```

Comment:
Should add an example as well, very hard to use otherwise.

Reply:
Yes, I added that in a new PR https://github.com/qlik-oss/nebula.js/pull/660